test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13575,6 +13577,22 @@ def create_script(self, body=None, uid=None, gid=None, hostname=None):
         if isinstance(body, list):
             body = '\n'.join(body)
 
+        if self.platform == 'cray' or self.platform == 'craysim':
+            body = body.split("\n")
+            for line in body:
```

Comment:
Since you need to find the index of line, use enumerate(): for i, line = enumerate(body)